Client/Frontend/Browser/Search/InitialSearchEngines.swift

```diff
@@ -25,6 +25,15 @@ class InitialSearchEngines {
                 case .ecosia: return "ecosia.org/opensearch"
             }
         }
+        
+        func excludedFromOnboarding(for locale: Locale) -> Bool {
```

Comment:
@kylehickinson removing it didn't last long :D
The product team want to keep Brave Search excluded from onboarding for all regions now